README.md

```diff
@@ -324,16 +351,17 @@ echo "gruntwork-io/terragrunt gruntwork-io/terratest" | git-xargs \
-| `--commit-message`       | The commit message to use when creating commits. If you supply this flag, but neither the optional `--pull-request-title` or `--pull-request-description` flags, then the commit message value will be used for all three.                                                                                                                                                                                                     | String  | No       |
-| `--skip-pull-requests`   | If you don't want any pull requests opened, but would rather have your changes committed directly to your specified branch, pass this flag. Note that it won't work if your Github repo is configured with branch protections on the branch you're trying to commit directly to!                                                                                                                                               | Boolean | No       |
+| Flag                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                    | Type    | Required |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | -------- |
+| `--branch-name`        | You must specify the name of the branch to make your local and remote changes on. You can further control branching behavior via `--skip-pull-requests` as explained below                                                                                                                                                                                                                                                     | String  | Yes      |
+| `--loglevel`           | Specify the log level of messages git-xargs should print to STDOUT at runtime. By default, this is INFO - so only INFO level messages will be visible. Pass DEBUG to see runtime errors encountered by your scripts or commands. Accepted levels are Trace, Debug, Info, Warning, Error, Fatal and Panic.                                                                                                                      | String  | Yes      |
```

Comment:
Here, we have `INFO` and `Info`. We should be consistent on capitalization.

Reply:
Thanks - fixed in cb00308